requirements.txt

```diff
@@ -23,6 +23,6 @@ future >= 0.18
 tqdm >= 4.66
 hyperopt >= 0.2.7
 python-graphviz >= 0.20
-scikit-learn >= 1.3.1
+scikit-learn == 1.3.2
```

Comment:
this pin looks to strict to me, is >=1,3,2,<1.4 better?

Reply:
Currently, there is no other version between 1.3.2 and 1.4.  I switched it to `1.3,1.4`